doc/examples/quick_start.py

```diff
@@ -92,7 +92,7 @@
 
 import matplotlib.pyplot as plt
 
-axial_middle = data.shape[2] / 2
+axial_middle = data.shape[2] // 2
```

Comment:
This line is the crucial change. 

The others are just eliminating un-needed whitespace